tests/integration_tests/data_sources/test_maria_ds.py

```diff
@@ -77,8 +77,8 @@ def test_maria_ds():
     def assert_expected_type(column_typing, expected_type, expected_subtype):
         assert column_typing['data_type'] == expected_type
         assert column_typing['data_subtype'] == expected_subtype
-        assert column_typing['data_type_dist'][expected_type] == 199
-        assert column_typing['data_subtype_dist'][expected_subtype] == 199
+        assert column_typing['data_type_dist'][expected_type] == 200
```

Comment:
creating a constant variable and referring to it everywhere else would remove confusion about where these numbers come from (`== 199`, `== 200`)

Reply:
everything else looks good